src/olympia/devhub/templates/devhub/versions/edit.html

```diff
@@ -34,9 +34,7 @@ <h3>{{ title }}</h3>
           {% if compat_form %}
           <tr>
             <th>{{ tip(_("Compatibility"),
-                       _("Adjusting application information here will allow users to install your "
-                         "add-on even if the install manifest in the package indicates that the "
-                         "add-on is incompatible.")) }}</th>
+                       _("Which versions of what applications your add-on is compatible with. It is not editable if you specified compatibility information in your manifest already.")) }}</th>
```

Comment:
Actually I was thinking of reverting this label entirely. It's only locked if you used `gecko_android`, which you should know if you used it... It's not a great place to add links because of the way we display the tooltip on hover only (it would disappear as you try to click on the link)

Reply:
b26ea73ccd363a587c1ab11812f04aa28e9ab6c3